wordpress-mobile/WordPress-Android
# Update GB to release 1.31.1

Update Gutenberg to release 1.31.1

To test:
 - Smoke test the writing flow in the Block Editor
 - Check the quote block  in dark mode and see that the vertical bar shows up correctly
 - Check the buttons block and see if adding multiple buttons and press Enter works correctly.

PR submission checklist:
